script.js

```diff
@@ -1,4 +1,98 @@
+//Base
+// Requirements
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+var gameDiceRoll = "Game Dice Roll";
+var diceSelect = "Dice Selection";
```

Comment:
Good descriptive names for the game mode variables. Dun think it's necessary to repeat it for the variables' values. 
i.e. the following will prob work as well:

var gameDiceRoll = 1;
var diceSelect = 2;